industrial_consumer.py

```diff
@@ -0,0 +1,66 @@
+import numpy as np
+import os
+from numpy.random import randint
+
+## IndustrialConsumer
+
+class IndustrialConsumer :
+
+    def __init__(self,demand,cost_elec_market):

```

Comment:
When we play with the manager, the industrial consumer is sent values of the demand at every stage t = 0..47 : future values of the demand should not be hosted in this class - and neither should costs, which may vary according to the behavior of all players.

I suggest

```suggestion
    def __init__(self):

```